### DISCUSSION TITLE
Is there a per-project .rc file like it was in Valet?

### DISCUSSION THREAD

**Discussion by MladenJanjetovic** [beyondcode/herd-community#381] [Q&A] [answered]
Hello,

I can't find this in documentation, nor here in git threads. But I am new in Heard. Has been using Valet for many years.
It will be useful to have per-project `.herdrc` or something like that, so it can be versioned and shared with team.

Thanks

**Comment by mpociot** [comment, marked as answer]
Hey 🙂
I hope you're doing well!

Right now we don't support that, but I want to add this to the next version

**Reply by MladenJanjetovic** [reply]
Hey man :)
Glad to hear from you! Hope you are well too!

Cool, I have like 20 laravel projects and all PHP versions from 7 to 8.3, so this was very useful in Valet.
Moving to Heard now. Great tool!

**Reply by markvaneijk** [reply]
It already works with `.valetrc` files :)

![CleanShot 2024-02-13 at 17 00 42@2x](https://github.com/beyondcode/herd-community/assets/1925388/71129bf6-4554-42cc-8057-e727889f8e9e)
